util/actions.js

```diff
@@ -0,0 +1,27 @@
+'use server';
+import { cookies } from 'next/headers';
+import { getFruitByName } from '../database/fruits';
+import { getCookieByName } from './cookies';
+
+export async function setFruitNote(formData) {
```

Comment:
this function name sounds too much like a useState setter - let's try to follow more of the naming in the Next.js docs

Reply:
https://github.com/upleveled/next-js-example-winter-2023-vienna-austria/blob/b185b14ff948fe87a99482d0aa749476c3c23826/app/fruits/%5BfruitId%5D/actions.js#L7